data/mods/Xedra_Evolved/recipes/inventor/explosive.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "portable_dark_age",
+    "using": [ [ "welding_standard", 10 ] ],
+    "qualities": [ { "id": "SAW_M", "level": 2 }, { "id": "SAW_M_FINE", "level": 1 }, { "id": "GRIND", "level": 1 } ],
+    "proficiencies": [
+      { "proficiency": "prof_metalworking", "time_multiplier": 3, "skill_penalty": 0 },
+      { "proficiency": "prof_welding_basic", "time_multiplier": 3, "skill_penalty": 0 },
+      { "proficiency": "prof_welding", "time_multiplier": 2, "skill_penalty": 0 },
+      { "proficiency": "prof_plasticworking", "time_multiplier": 2, "skill_penalty": 0 }
+    ],
+    "components": [
+      [ [ "cable", 3 ] ],
+      [ [ "scrap_aluminum", 5 ] ],
+      [ [ "e_scrap", 2 ] ],
+      [ [ "power_supply", 4 ] ],
+      [ [ "magnetron", 4 ] ],
+      [ [ "inventor_medium_cell", 1 ] ]
+    ],
+    "time": "7 h",
+    "skill_used": "deduction",
+    "difficulty": 5,
+    "skills_required": [ "electronics", 3 ],
+    "flags": [ "SECRET" ],
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_ARMOR"
```

Comment:
An EMP bomb doesn't seem like armor to me, except in the "the best defense is a good offense" sense.